### PR TITLE
[ra2 ch1] nile proofreading

### LINKED ISSUE #3266: [RA2 Ch1] Nile Proofreading

### PR DESCRIPTION
**Pull request by rgstori**
fixes #3266

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter01.rst [hunk 1]
```diff
@@ -27,38 +27,57 @@ workloads". For example, start the Reference Architecture from a vanilla Kuberne
 provide clear evidence that a functional requirement cannot be met by that system (say, multi-NIC support), only then
 the RA would add the least invasive, Kubernetes-community aligned extension (say, Multus) to fill the gap. If there are
 still gaps that cannot be filled by standard Kubernetes community technologies or extensions then the RA will concisely
-document the requirement and approach the relevant project maintainers with a request to add this functionality into
-the feature set.
+document the requirement in the :doc:`"Gaps, Innovation, and Development"<chapters/chapter07>` chapter of this document
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter01.rst:30]: this should use :ref:`chapters/chapter07
```suggestion
document the requirement in the :ref;:chapters/chapter07` chapter of this document
```

**Review by pgoyal01 — APPROVED** [review]
subject to a change

**Review by steelescotr — APPROVED** [review]
Approval conditional on addressing @pgoyal01 comments

File: doc/ref_arch/kubernetes/chapters/chapter01.rst (hunk 1 above)
Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter01.rst:30]: @pgoyal01 thanks - although I think ref only works for links to a label within the document, for linking to a whole page (like ch4 in this case), only doc works - cf https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Pull request merged**